app/jobs/purge_job.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+##
+# job to purge unpublished objects
+class PurgeJob < GenericJob
+  queue_as :default
+
+  def perform(bulk_action_id, params)
+    super
+
+    with_bulk_action_log do |log_buffer|
+      log_buffer.puts("#{Time.current} Starting #{self.class} for BulkAction #{bulk_action_id}")
+      update_druid_count
+
+      pids.each do |current_druid|
+        log_buffer.puts("#{Time.current} #{self.class}: Attempting to index #{current_druid} (bulk_action.id=#{bulk_action_id})")
```

Comment:
Should this log statement be something like "Attempting to purge" instead of "to index"?